extensions/wikia/Chat2/Chat_setup.php

```diff
@@ -155,6 +157,7 @@
 define( 'CUC_TYPE_CHAT', 128 );    // for CheckUser operation type
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)